### PR TITLE
docs: add otel metrics collector documentation

### PR DESCRIPTION
**Pull request by swiatekm-sumo**
### Checklist

- [ ] Changelog updated or skip changelog label added
- [x] Documentation updated
- [ ] Template tests added for new features
- [ ] Integration tests added or modified for major features

### CODE REVIEW

File: docs/opentelemetry-collector/metrics.md [hunk 1]
```diff
@@ -1,8 +1,77 @@
...
+The Otel metrics collector doesn't support recording rules. As the collector doesn't actually maintain a database of time series the way
+Prometheus does, it cannot easily do aggregations over the whole dataset, or over longer time periods.
+
+> [!WARNING]  
+> Some of the Node-specific panels of the Sumo Kubernetes App will not work as a result of this. An update to the App will be released in
+> the near future.
+
+#### No extended features of the Prometheus ecosystem
+
+OpenTelemetry collector is just a collector - it scrapes metrics data, transforms it, and forwards it to Sumo. It does not maintain a
```
Comment (sumo-drosiek) [inline docs/opentelemetry-collector/metrics.md:64]: Please link `values.yaml`

**Review by sumo-drosiek — APPROVED** [review]
LGTM

File: docs/opentelemetry-collector/metrics.md (hunk 1 above)
Comment (sumo-drosiek) [reply docs/opentelemetry-collector/metrics.md:64]: Its about

```
Configuration keys for the Otel metrics collector live under the `sumologic.metrics.collector` section of the values.yaml file.
Configuration for ServiceMonitors elsewhere in the Chart will still apply.
```
not sure why co,mment has been added here :confused:

Comment (swiatekm-sumo) [reply docs/opentelemetry-collector/metrics.md:64]: Done.

**Pull request merged**